AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "galleryItemName",
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.gallery.providers/galleryItems": {
+      "get": {
+        "tags": [
+          "ProviderGalleryRegistrations"
+        ],
+        "description": "Lists gallery registrations",
+        "operationId": "ProviderGalleryRegistrations_List",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ResourceDefinition"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+     "GalleryItemUriPayload": {
+      "description": "Gallery item URI, Points to the http or https uri where the galery package is located, typically a blob uri",
+      "type": "object",
+      "properties": {
+        "GalleryItemUri": {
+          "description": "Gets or sets the gallery item URI.",
+          "type": "string"
+        }
+      }
+    },
+    "ResourcePlan": {
+      "type": "object",
+      "description": "The reosurce plan object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The name of the plan",
+          "readOnly": true
+        },
+        "Product": {
+          "type": "string",
+          "description": "The product code of the plan",
+          "readOnly": true
+        },
+        "PromotionCode": {
+          "type": "string",
+          "description": "The promotion code of the plan",
+          "readOnly": true
+        },
+        "Publisher": {
+          "type": "string",
+          "description": "The publisher of the plan",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceSku": {
+      "type": "object",
+      "description": "The resource sku object",
+      "properties": {
+        "Name": {
```

Comment:
Sku has properties that can be an enum. Please describe them as an enum if they fall in that category.  Please take a look at the x-ms-enum extension inside documentation/swagger-extensions.md file of this repo.